lib/ansible/plugins/httpapi/fortios.py

```diff
@@ -65,7 +66,11 @@ def set_become(self, become_context):
     def login(self, username, password):
         """Call a defined login endpoint to receive an authentication token."""
 
-        data = "username=" + urllib.parse.quote(username) + "&secretkey=" + urllib.parse.quote(password) + "&ajax=1"
+        if sys.version_info[0] >= 3:
```

Comment:
Again, you don't need to do _any_ of this. Just put it back to the way it was, `six` will make sure that the right version of `quote()` gets used.

Reply:
Yes, right. Sorry I added the change in a second commit bcs I forgot to include it in the first one. I guess it is ok now (I left the version containing 'urllib.parse.quote')